truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/impl/Accessor.java

```diff
@@ -114,6 +114,11 @@ protected Object evalInContext(Source source, Node node, MaterializedFrame mFram
         } catch (ClassNotFoundException ex) {
             throw new IllegalStateException(ex);
         }
+        try {
+            Class.forName("com.oracle.truffle.api.debug.Debugger", true, Accessor.class.getClassLoader());
+        } catch (ClassNotFoundException ex) {
+            throw new IllegalStateException(ex);
+        }
```

Comment:
Can you please report an issue and give me steps to reproduce the problem? I tried:
```diff
diff --git a/truffle/com.oracle.truffle.api.test/src/com/oracle/truffle/api/vm/ArrayTruffleObject.java b/truffle/com.oracle.truffle.api.test/src/com/oracle/truffle/api/vm/ArrayTruffleObject.java
index 1ccc65f..12f0d68 100644
--- a/truffle/com.oracle.truffle.api.test/src/com/oracle/truffle/api/vm/ArrayTruffleObject.java
+++ b/truffle/com.oracle.truffle.api.test/src/com/oracle/truffle/api/vm/ArrayTruffleObject.java
@@ -34,8 +34,22 @@ import com.oracle.truffle.api.interop.ForeignAccess;
 import com.oracle.truffle.api.interop.Message;
 import com.oracle.truffle.api.interop.TruffleObject;
 import com.oracle.truffle.api.nodes.RootNode;
-
-final class ArrayTruffleObject implements TruffleObject, ForeignAccess.Factory10 {
+import com.oracle.truffle.api.source.Source;
+import java.io.IOException;
+import org.junit.Test;
+
+public final class ArrayTruffleObject implements TruffleObject, ForeignAccess.Factory10 {
+    public ArrayTruffleObject() {
+        this(null, null);
+    }
+    @Test
+    public void x() {
+        try {
+            PolyglotEngine.newBuilder().build().eval(Source.fromText("return x", "empty").withMimeType("application/x-test-import-export-2"));
+        } catch (IOException ex) {
+            throw new IllegalStateException(ex);
+        }
+    }
     private final ForeignAccess access;
     private final Object[] values;
     private final Thread forbiddenDupl;

```
and `mx unittest ArrayTruffleObject` works fine.